### ISSUE TITLE
Add mouseWheelListener direction option for WebSlider

### ISSUE THREAD

**Issue by nmagre** [mgarin/weblaf#242]
I think **mouseWheelListener** in **WebSlider** should be inverted by default ( "-" instead of "+" e.getWheelRotation()" ), when we go up for most of people, it's the same as forward and not  backward.  

For the moment I remove your listener and add my own but maybe you can add a simple function "**invertMouseWheelDirection(boolean invert)**" to change direction.

**Comment by mgarin** [comment]
Good point. I have checked some other usage cases - WebLaF indeed uses incorrect (inverted) slide direction by default which is not too convenient.

I have changed that and also provided an option to change direction if needed:
```java
slider.setInvertMouseWheelDirection ( true );
```
By default it is set to false.

This change is only available in source code form so far, but I will be releasing updated pre-release artifacts later today and this change will be included there.
